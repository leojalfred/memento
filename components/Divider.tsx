import { View } from 'react-native'

export default function Divider() {
  return <View className="h-full bg-gray-700" />
}
