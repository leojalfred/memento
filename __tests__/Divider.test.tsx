import Divider from '@/components/Divider'
import { render } from '@testing-library/react-native'

describe('<Divider />', () => {
  it('renders correctly', () => {
    const tree = render(<Divider />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
