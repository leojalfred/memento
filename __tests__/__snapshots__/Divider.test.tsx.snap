// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Divider /> renders correctly 1`] = `
<View
  className="h-[27.3px] w-px bg-gray-700 dark:bg-gray-300"
/>
`;
